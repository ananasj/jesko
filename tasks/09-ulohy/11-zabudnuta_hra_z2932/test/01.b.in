7
17 22 34 32 14 3 4
1 35 36 6 12 18 26
45 11 19 25 38 16 29
37 2 42 44 9 31 24
8 41 21 0 46 20 39
40 13 27 7 10 47 30
48 15 23 43 5 28 33
13
48
25
46
25
16
34
34
44
5
4
43
22
17
