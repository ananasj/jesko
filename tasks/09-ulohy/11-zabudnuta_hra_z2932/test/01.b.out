17 22 34 32 14 3 4
1 35 36 6 12 18 26
45 11 19 25 38 16 29
37 2 42 44 9 31 24
8 41 21 46 0 20 39
40 13 27 7 10 47 30
48 15 23 43 5 28 33
